elyra/contents/tests/test_handlers.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright 2018-2021 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import pytest
+import json
+
+from tornado.httpclient import HTTPClientError
+from .test_utils import expected_http_error, expected_response, expected_response_empty
+
+
+async def test_file_not_found(jp_fetch):
+    filepath = "nofile.py"
+
+    with pytest.raises(HTTPClientError) as e:
+        await jp_fetch('elyra', 'contents/properties', filepath)
+    assert expected_http_error(e, 404)
+
+
+async def test_file_is_not_directory(jp_fetch, create_directory, directory_name):
+    with pytest.raises(HTTPClientError) as e:
+        await jp_fetch('elyra', 'contents/properties', 'dir.py')
+    assert expected_http_error(e, 400)
+
+
+async def test_invalid_post_request(jp_fetch):
+    response = await jp_fetch('elyra', 'contents/properties', 'dir.py', body=json.dumps(""), method='POST')
```

Comment:
Same: `'dir.py'` -> `directory_name`?  Will need to add the fixture name to the test method.